esy-lib/Tarball.re

```diff
@@ -87,6 +87,19 @@ let unpackWithTar = (~stripComponents=?, ~dst, filename) => {
   | Some(stripComponents) =>
     Fs.withTempDir(out => {
       let* () = unpack(out);
+      let normalizedOut = EsyBash.normalizePathForCygwin(Path.show(out));
+      let normalizedOut =
+        Fpath.of_string(normalizedOut) |> Stdlib.Result.get_ok;
+      let* _ =
+        Fs.traverse(
+          ~f=
+            (path, _) => {
+              // Logs.app(m => m("%s\n", Fpath.to_string(path)));
+              Unix.chmod(Fpath.to_string(path), 0o777);
```

Comment:
Maybe be opt for Lwt's chmod to remain consistent. 
Side note: Mixing of Bos and Lwt is confusing. We need to address this someday. Unrelated to this PR